Doc/library/othergui.rst

```diff
@@ -30,8 +30,11 @@ available for Python:
       for generating bindings for C++ libraries as Python classes, and
       is specifically designed for Python.
 
-   `PySide <https://wiki.qt.io/PySide>`_
-      PySide is a newer binding to the Qt toolkit, provided by Nokia.
+   `PySide2 <https://wiki.qt.io/PySide>`_
```

Comment:
This seems to redirect to https://wiki.qt.io/Qt_for_Python. (Maybe a better page is https://doc.qt.io/qtforpython/?)

Reply:
There were some changes since I wrote this patch. I agree that now that the docs are online it's a better link.